identity.go

```diff
@@ -110,6 +110,45 @@ func (i *IdentityIterator) Next() bool {
 	return true
 }
 
+// WriteTo encodes and writes all IdentityInfos to w.
+// It returns the number of bytes written to w and
+// the first error encounterred, if any.
+func (i *IdentityIterator) WriteTo(w io.Writer) (int64, error) {
+	type Response struct {
+		Identity  Identity  `json:"identity"`
+		Policy    string    `json:"policy,omitempty"`
+		CreatedAt time.Time `json:"created_at,omitempty"`
+		CreatedBy Identity  `json:"created_by,omitempty"`
+
+		Err string `json:"error,omitempty"`
+	}
+	if i.err != nil || i.closed {
+		return 0, i.err
+	}
```

Comment:
Will return `0, nil` if closed.  Maybe something like:

```suggestion
	if i.err != nil {
		return 0, i.err
	} else if i.closed != nil {
		return 0, errors.New("writeTo called after Close")
	}

```